common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -28,23 +34,71 @@ public class MLModel implements ToXContentObject {
     public static final String OLD_MODEL_CONTENT_FIELD = "content";
     public static final String MODEL_CONTENT_FIELD = "model_content";
 
+    public static final String DESCRIPTION_FIELD = "description";
+    public static final String MODEL_FORMAT_FIELD = "model_format";
+    public static final String MODEL_STATE_FIELD = "model_state";
+    public static final String MODEL_CONTENT_SIZE_IN_BYTES_FIELD = "model_content_size_in_bytes";
+    public static final String MODEL_CONTENT_HASH_FIELD = "model_content_hash";
```

Comment:
This field isn't self explanatory to me.

Reply:
Do you have any suggestion? Basically this field is to track the model file's SHA256 hash value.